go/libkb/pipeowner_windows.go

```diff
@@ -85,10 +86,21 @@ func IsPipeowner(name string) (bool, error) {
 		if err2 != nil {
 			return false, err // return original busy error
 		}
-		fileSid, err = GetFileUserSid(name)
+		pipeSid, err = GetFileUserSid(name)
 	}
 	if err != nil {
 		return false, err
 	}
-	return windows.EqualSid(fileSid, userSid), nil
+	isEqual := windows.EqualSid(pipeSid, userSid)
```

Comment:
Can you throw some more debugging in right here too?

Reply:
The caller always prints out that error someplace, do you want it debug logged here too, or...?